examples/apps/attributable-map/package.json

```diff
@@ -49,21 +47,16 @@
 		"@fluidframework/build-tools": "^0.34.0",
 		"@fluidframework/eslint-config-fluid": "^4.0.0",
 		"@fluidframework/test-tools": "^1.0.195075",
-		"@types/jest": "29.5.3",
+		"@types/jest-environment-puppeteer": "workspace:",
```

Comment:
delete instead?
@types/node dependency is also suspect without any testing.
And just below that cross-env appears unused. And there are two more puppeteer deps remain.